src/main/java/com/team4/testingsystem/entities/FileAnswer.java

```diff
@@ -0,0 +1,81 @@
+package com.team4.testingsystem.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "file_answer")
+public class FileAnswer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne
+    @JoinColumn(name = "id", insertable = false, updatable = false)
+    private Question question;
+
+    @Column(name = "url")
+    private String url;
+
+    public FileAnswer() {
+    }
+
+    public static Builder newBuilder() {
+        return new FileAnswer().new Builder();
+    }
+
+    public class Builder {
+        private Builder() {
+        }
```

Comment:
Again, we have two different builder implementations)
It seems that we've chosen [this implementation](https://github.com/AntonMaksimenkau/team4_backend/blob/develop/src/main/java/com/team4/testingsystem/entities/Question.java#L101), without private constructors and so on.